src/main/java/com/google/maps/GeocodingApiRequest.java

```diff
@@ -105,8 +105,8 @@ public GeocodingApiRequest bounds(LatLng southWestBound, LatLng northEastBound)
    * parameter will only influence, not fully restrict, results from the geocoder.
    *
    * <p>For more information see <a
-   * href="https://developers.google.com/maps/documentation/geocoding/?hl=pl#RegionCodes">Region
-   * Codes</a>.
+   * href="https://developers.google.com/maps/documentation/geocoding/intro?hl=pl#RegionCodes">Region
```

Comment:
Similar to my previous comment, the URL should be:
https://developers.google.com/maps/documentation/geocoding/intro#RegionCodes